coderabbit.yml

```
@@ -107,14 +107,117 @@ reviews:
+        - Ensure efficient event handler patterns
+
+        **React Patterns:**
+        - Verify proper hook usage and custom hook patterns
+        - Check for correct error boundary implementation
+        - Validate loading and error state handling
+        - Ensure proper form handling with react-hook-form patterns
+
+    - path: "src/hooks/**/*.ts"
+      instructions: |
+        Review custom hooks for React and TanStack Query patterns:
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)